src/main/java/fr/insee/survey/datacollectionmanagement/contact/controller/AddressController.java

```diff
@@ -30,9 +29,6 @@
 import java.util.List;
 
 @RestController
-@PreAuthorize("@AuthorizeMethodDecider.isInternalUser() "

```

Comment:
normal to not have preauthorize in new version ?

Reply:
oups